tests/conftest.py

```diff
@@ -7,7 +7,13 @@
 
 
 @pytest.fixture(scope='module')
-def client() -> Flask:
+def app() -> Flask:
     app = create_app()
+    app.config.update({'TESTING': True})
```

Comment:
what does this do?

Reply:
I'm sure this does all kinds of magic with requests for testing purposes. All I know is that the Flask docs recommend setting it. It removes the need for a separate test config.

Reply:
TESTING
Enable testing mode. Exceptions are propagated rather than handled by the the app’s error handlers. Extensions may also change their behavior to facilitate easier testing. You should enable this in your own tests.

Default: False

Reply:
Seems beneficial, at least for unit testing; we'll check later on about API testing if it affects the custom error responses (it doesn't seem so).

Reply:
This can be done using `with pytest.raises(ExceptionName):`

Reply:
Test already exist for custom errors in this repo. They seem to still work ok.